Rubberduck.CodeAnalysis/Inspections/Concrete/ExcessiveInterfaceMembersInspection

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Linq;
+using Rubberduck.CodeAnalysis.Inspections.Abstract;
+using Rubberduck.Common;
+using Rubberduck.Parsing.Annotations;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.Parsing.VBA.DeclarationCaching;
+using Rubberduck.Resources.Inspections;
+
+namespace Rubberduck.CodeAnalysis.Inspections.Concrete {
+
+    internal sealed class ExcessiveInterfaceMembersInspection : DeclarationInspectionBase {
+
+        public ExcessiveInterfaceMembersInspection(IDeclarationFinderProvider declarationFinderProvider, params DeclarationType[] relevantDeclarationTypes) 
+            : base(declarationFinderProvider, relevantDeclarationTypes) {
+        }
+
+        public ExcessiveInterfaceMembersInspection(IDeclarationFinderProvider declarationFinderProvider, DeclarationType[] relevantDeclarationTypes, DeclarationType[] excludeDeclarationTypes) 
+            : base(declarationFinderProvider, relevantDeclarationTypes, excludeDeclarationTypes) {
+        }
+
+        protected override bool IsResultDeclaration(Declaration declaration, DeclarationFinder finder) {
+
+            if (IsInterfaceDeclaration(declaration)) {
+                var module = (ClassModuleDeclaration)declaration;
+                var pub = module.Members.Where(member => member.Accessibility == Accessibility.Public); //note: unsure if this handles property get/let/set correctly - doesnt account for redundancy and what needs to be counted
```

Comment:
Both `Global`, deprecated but equivalent to `Public`, and `Implicit` do the same for module level members.